students/hyska_monika/lesson_01_basics/AppleSharing.py

```diff
@@ -0,0 +1,12 @@
+# This program returns:
+# How many apples will each single student get?
+# How many apples will remain in the basket?
+
+n = int(input("Numbers of students: "))
+k =  int(input("Numbers of apples: "))
+
+n_apples_s = k // n #numbers apples for 1 student
+n_apples_b = k % n #numbers apples left on basket
```

Comment:
`apples_per_student` `apples_left` or `apples_basket` would be better names. Try to avoid single letters.